src/playermat/ClueCounter.ttslua

```diff
@@ -67,20 +65,8 @@ function findValidItemsInSphere()
   return retval
 end
 
-function removeAllClues()
```

Comment:
The changes made here to the clue clicker break it; specifically when switching between clue tokens to clue clicker, the clue tokens fail to delete and throw an error: 
`function <removeAllClues>: cannot convert a table to a clr type LuaGameObjectScript`

Reply:
The new changes do fix the behaviour, but their is a new error thrown now:

`function <onClick_toggleOption>: attempt to index nil value`